src/static/js/routes/careers/working-at-cfpb/index.js

```diff
@@ -0,0 +1,21 @@
+/* ==========================================================================
```

Comment:
Why do page named initializing instead of just looking for the `.js-mobile-carousel` class and using the same init on all the pages that include it (it doesn't seem they're any different)?